File: Mobile Application/frontend/lib/models/model_output.dart
class ModelOutput {
  final double doublePlanter;
  final double weedCluster;
  final double standingWater;

  ModelOutput({
    required this.doublePlanter,
    required this.weedCluster,
    required this.standingWater,
  });

  factory ModelOutput.fromJson(Map<String, dynamic> json) {
    return ModelOutput(
      doublePlanter: json['double_planter'],
      weedCluster: json['weed_cluster'],
      standingWater: json['standing_water'],
    );
  }
}

File: Mobile Application/backend/models/ModelOutput.js
import mongoose from 'mongoose';

const ModelOutputSchema = new mongoose.Schema({
  double_planter: { type: Number, required: true },
  weed_cluster: { type: Number, required: true },
  standing_water: { type: Number, required: true },
  updatedAt: { type: Date, default: Date.now },
});

export default mongoose.model('ModelOutput', ModelOutputSchema);
